student-handbook.md

```diff
@@ -11,21 +28,4 @@ We recommend you be familiar with these tools and technologies:
 3. [Web accessibility](/accessibility/) (15 minutes)
```

Comment:
I thought we were removeing all of the time estimations.

Reply:
We are: I hadn't got to this section yet.